common/k8stest/util.go

```diff
@@ -480,6 +480,112 @@ func MayastorReady(sleepTime int, duration int) (bool, error) {
 	return ready, err
 }
 
+// MayastorReadyWithException checks if the product installation is ready
+// with exception for excluded deployments, daemonsets and statefulsets, deployments and daemonsets
```

Comment:
deployments and daemonsets repeated twice in the comment

Reply:
updated the comment